code/game/objects/structures/girders.dm

```diff
@@ -30,9 +30,9 @@
 
 	if(reinf_material)
 		playsound(src.loc, 'sound/items/Screwdriver.ogg', 100, 1)
-		visible_message(SPAN_NOTICE("\The [user] begins unscrewing \the [reinf_material.solid_name] struts from \the [src]."))
+		visible_message(SPAN_NOTICE("\The [user] begins unscrewing \the [reinf_material.solid_name] rods from \the [src]."))
```

Comment:
while we're at it these might need to be `reinf_material.adjective_name` so that you'd get e.g. "oaken rods" rather than "oak rods". it's not a huge deal though because usually both are valid, but i like using the adjective form where possible

edit: this is an issue in more places than just here, i will do a pass on it at some point